### DISCUSSION TITLE
Help fixing everforest colorscheme

### DISCUSSION THREAD

**Discussion by synic** [Bekaboo/dropbar.nvim#148] [General]
Love dropbar! There's a couple themes that don't display it quite right and I'd like to fix it, however, I don't really know where to start.

The main problem is that the background is black for active windows. I looks fine for inactive windows:

![image](https://github.com/Bekaboo/dropbar.nvim/assets/30906/9dbf2a18-4387-4c00-a844-31c4addde316)

What is the group name for the background there? I just want to make it the same background as regular text

**Comment by Bekaboo** [comment]
Hi, thanks for reporting this issue! It's a duplicate of https://github.com/Bekaboo/dropbar.nvim/issues/118, TL;DR, you need to change hl-WinBar, which is different from the default normal background in the new default colorscheme adapted by neovim nightly (0.10+). I recommend using the following snippet to avoid the background color issue no matter what colorscheme you are using:

```lua
---Set WinBar & WinBarNC background to Normal background
---@return nil
local function clear_winbar_bg()
  ---@param name string
  ---@return nil
  local function _clear_bg(name)
    local hl = vim.api.nvim_get_hl(0, { name = name, link = false })
    if hl.bg or hl.ctermbg then
      hl.bg = nil
      hl.ctermbg = nil
      vim.api.nvim_set_hl(0, name, hl)
    end
  end

  _clear_bg('WinBar')
  _clear_bg('WinBarNC')
end

clear_winbar_bg()

vim.api.nvim_create_autocmd('ColorScheme', {
  group = vim.api.nvim_create_augroup('WinBarHlClearBg', {}),
  callback = clear_winbar_bg,
})
```

**Reply by synic** [reply]
Yup, this sorts it out. Thank you very much!
